sechub-wrapper-xray/src/test/java/com/mercedesbenz/sechub/wrapper/xray/api/XrayAPIResponseFactoryTest.java

```diff
@@ -14,21 +14,24 @@
 import org.mockito.Mock;
 import org.mockito.Mockito;
 
-import com.mercedesbenz.sechub.wrapper.xray.cli.XrayWrapperRuntimeException;
+import com.mercedesbenz.sechub.wrapper.xray.XrayWrapperException;
 
-class XrayAPIResponseBuilderTest {
+class XrayAPIResponseFactoryTest {
 
     @Mock
     HttpURLConnection connection;
     ByteArrayInputStream inputStream;
 
+    XrayAPIResponseFactory xrayAPIResponseFactory;
+
     @BeforeEach
     void beforeEach() {
         connection = mock(HttpURLConnection.class);
+        xrayAPIResponseFactory = new XrayAPIResponseFactory();
     }
 
     @Test
-    void getHttpResponseFromConnection_get_valid_http_response() throws IOException {
+    void getHttpResponseFromConnection_get_valid_http_response() throws IOException, XrayWrapperException {
```

Comment:
Please rename test methods like "getHttpResponse..." to something like "createHttpResponse..." or use a name which does not contain the caller name (but currently the get is irritating if somebody does not know the old method names...)